### PR TITLE
Crowdstrike integration

### PR DESCRIPTION
**Pull request by shishkinnico**
# What changed

https://deploy-preview-1138--logz-docs.netlify.app/shipping/security-sources/crowdstrike.html

New document on Crowdstrike - SIEM integration

----

## Pages to review

- LinkToPage1
- LinkToPage2

## Remaining work

- [ ] Technical review
- [ ] Copy Review
- [ ] Redirects: 

## Post launch

To be completed by the docs team upon merge:

- [ ] Teams to update with the new information:
- [ ] Replace original content on the support portal with 'this page has been moved to ...' - paste the URL
- [ ] Update these log shipping pages in the app: - paste the URL

### CODE REVIEW

**Review by shishkinnico — COMMENTED** [review]
Typo corrected

**Review by shishkinnico — COMMENTED** [review]
Style correction

File: _source/logzio_collections/_security-sources/crowdstrike.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+
+<div class="tasklist">
+
+
+##### Configure Crowdstrike connector
+
+
+1. Open the configuration file located at `/opt/crowdstrike/etc/cs.falconhoseclient.cfg`.
+2. Enter the Client ID value into the **client_id** value field.
+3. Enter the Client Secret value into the **client_secret** value field.
```
Comment (mirii1994) [inline _source/logzio_collections/_security-sources/crowdstrike.md:66]:
```suggestion
2. Enter the Client ID value into the `client_id` value field.
3. Enter the Client Secret value into the `client_secret` value field.
```

Maybe we should do it like this?

File: _source/logzio_collections/_security-sources/crowdstrike.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+4. Make sure that the base URL in `api_url` and `request_token_url` corresponds to the base URL for your account.
```
Comment (mirii1994) [inline _source/logzio_collections/_security-sources/crowdstrike.md:67]:
```suggestion
4. Make sure that the **base URL** in `api_url` and `request_token_url` corresponds to the base URL for your account.
```

File: _source/logzio_collections/_security-sources/crowdstrike.md [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+     @type tail
+     path /var/log/crowdstrike/falconhoseclient/output
+     pos_file /var/log/td-agent/falconhoseclient.log.pos
+     tag crowdstrike-fluentd
+     <parse>
+       @type none
+     </parse>
+   </source>
+   ​
+   # Concating the event as one log
```
Comment (mirii1994) [inline _source/logzio_collections/_security-sources/crowdstrike.md:99]:
```suggestion
   # Concatenating the event as one log
```
My bad, typo :)

**Review by mirii1994 — CHANGES_REQUESTED** [review]
Amazing job! please see my notes :)

**Comment by shishkinnico** [comment]
@mirii1994 Thank you for your comments, makes total sense, I've updated it.

**Review by YajB — APPROVED** [review]

**Pull request merged**